R/functions.R

```diff
@@ -470,7 +542,7 @@ m.auc <- function(data.m, group.v) {
 #' @return A list with the cell types and their differentially expressed genes
 #' 
 #' @importFrom MAST FromFlatDF zlm lrTest
-#' @importFrom reshape melt
+#' @importFrom data.table melt //TODO discuss how to define new packages
```

Comment:
rather than reshape i used data.table for all melt operations, this seems faster

Reply:
should be no problem, just add it to the DESCRIPTION file, similar to the other packages.
Is `reshape` still used anywhere else in the package or can it then be removed?

Reply:
reshape is not used anymore, can be removed